ale_linters/php/psalm.vim

```diff
@@ -0,0 +1,38 @@
+" Author: richard marmorstein <https://github.com/twitchard>
```

Comment:
Add some tests for the command and executable here. See `:help ale-development`.